src/components/DMsList/DMsList.jsx

```diff
@@ -1,45 +1,39 @@
 import SimpleBar from 'simplebar-react';
-import {
-  Wrapper,
-  Text,
-  ListItems,
-  Item,
-  Avatar,
-  ChatName,
-  MessageDay,
-} from './DMsListStyled';
+import { Item, Avatar, ChatName, MessageDay } from './DMsListStyled';
+import { ListStyled, Text, ListItems } from '../RoomsList/RoomsListStyled.js';
 import 'simplebar-react/dist/simplebar.min.css';
 
 const DMsList = () => {
-  const chats = [];
+  const chats = ['Anya', 'Vasya'];
 
   return (
-    <Wrapper>
-      {chats.length === 0 && (
+    <ListStyled>
+      {chats?.length ? (
+        <ListItems>
+          <SimpleBar style={{ maxHeight: 570 }}>
+            {chats.map(name => (
+              <Item key={name}>
+                <Avatar>
+                  <img src="../../../public/img/Avatar.png" alt="avatar" />
```

Comment:
Use file import instead. 
import avatarImage from '../../../public/img/Avatar.png';


```suggestion
                  <img src={avatarImage} alt="avatar" />
```